test/ShapeCrawler.Tests.Unit/TextFrameTests.cs

```diff
@@ -505,5 +505,134 @@ public void BottomMargin_getter_returns_bottom_margin_of_text_frame_in_centimete
             // Assert
             bottomMargin.Should().Be((decimal)expectedMargin);
         }
+
+        [Test]
+        public void SlideNotes_getter_returns_notes()
```

Comment:
I think the more suitable place for these tests is `SlideTests.cs` with naming `Notes_Getter_returns_notes()`.

Reply:
Done in latest